events/2024.11.Munich/VPN_static_addr.md

```diff
@@ -0,0 +1,15 @@
+# IP address allocation table for Plugfest VPN
+
+In plugfest VPN network, the DHCP server dynamically allocates IP addresses between 192.168.30.128 to 250.  
+If you would like to use statically allocated IP addresses, please secure your addresses
+by updating this markdown file and making a pull request.
+
+| IP Address | Description |
+| --- | --- |
+|192.168.30.1       | Gateway         |
+|192.168.30.2-9     | Reserved        |
+|192.168.30.10      | free            |
+| ...               | ...             |
+|192.168.30.127     | free                  |
```

Comment:
I guess these will be used by the devices brought to plugfest?

Reply:
Yes, that's right. The IP addresses from 10 to 127 are intended to be used by the Thing, Consumer, Discoverer, etc. in the plugfest.

Reply:
Ok I see. Resolving